paymentsheet/src/main/java/com/stripe/android/paymentsheet/IntentConfirmationInterceptorKtx.kt

```diff
@@ -27,8 +32,26 @@ internal suspend fun IntentConfirmationInterceptor.intercept(
                 requiresSaveOnConfirmation = paymentSelection.requiresSaveOnConfirmation,
             )
         }
+        null -> {
```

Comment:
This case is expected in flow controller.

If the merchant doesn't disable the buy button, and calls confirm with nothing selected, it would have crashed before.